### PR TITLE
[TF-11102] Added history component

### PR DESCRIPTION
**Pull request by mocarz**

### CODE REVIEW

File: sandbox-sdk-app/src/app/components/call-history/call-history.component.html [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+                    >north_west</mat-icon
+                >
+    
+                <div mat-line>
+                    <b>{{ historyItem.name }}</b> #{{ historyItem.id }}
+                </div>
+    
+                <div mat-line>
+                    Status: <b>{{ historyItem.status }}</b
+                    >, is child call: <b>{{ !!historyItem?.parentId }}</b>, phone number:
```
Comment (Ventrae) [inline sandbox-sdk-app/src/app/components/call-history/call-history.component.html:35]: You might want to not break the tag between two lines like this

File: sandbox-sdk-app/src/app/components/call-history/call-history.component.ts [hunk 2]
```diff
@@ -0,0 +1,26 @@
+import { Component } from '@angular/core';
+import { of } from 'rxjs';
+import { switchMap } from 'rxjs/operators';
+import { ConnectorService } from 'src/app/services/connector.service';
+
+
+@Component({
+  selector: 'app-call-history',
+  templateUrl: './call-history.component.html',
+  styleUrls: ['./call-history.component.scss']
```
Comment (Ventrae) [inline sandbox-sdk-app/src/app/components/call-history/call-history.component.ts:10]: I can see inconsistency between selector (**app-call-history**) and everything else like class name (**CallHistory**Component), components html, css, ts files names (**call-history**.component). Not sure if that's intentional but you might want to shorten it to just `call-history` selector.

**Comment by Ventrae** [comment]
Also, can `sandbox-sdk-app/src/app/components/call-history/call-history.component.scss` file be removed if it's empty?

File: sandbox-sdk-app/src/app/components/call-history/call-history.component.ts (hunk 2 above)
Comment (mocarz) [reply sandbox-sdk-app/src/app/components/call-history/call-history.component.ts:10]: @PabloDotcom, you have added `app-` prefix to every component's selector in the application. Can it be safely removed or there is a good reason for it?

File: sandbox-sdk-app/src/app/components/call-history/call-history.component.html (hunk 1 above)
Comment (mocarz) [reply sandbox-sdk-app/src/app/components/call-history/call-history.component.html]: Fixed

File: sandbox-sdk-app/src/app/components/call-history/call-history.component.ts (hunk 2 above)
Comment (PabloDotcom) [reply sandbox-sdk-app/src/app/components/call-history/call-history.component.ts:10]: `app-` prefix should be there. It's anti-pattern to not use that.

**Comment by PabloDotcom** [comment]
Leave empty call-history.component.scss. It's not a problem to have empty file with style and reference in *.ts component file. In future you can easily add style code without adding a file. Structure of project is unified. 
Anyway, it won't be in final bundle...

**Review by PabloDotcom — APPROVED** [review]

**Review by louishawkins — APPROVED** [review]

**Pull request merged**